### DISCUSSION TITLE
[Box] Display prop and storybook

### DISCUSSION THREAD

**Discussion by aweell** [dxc-technology/halstack-react#568] [Q&A] [answered]
How the prop currently works accepts any `display` CSS property as a string, this is problematic since there are 17 (Let's remove the two-value syntax since it doesn't have full support yet...) possible values for that property.

* Should we contemplate in the `box.stories.jsx` all possible CSS `display` values?
* Should we restrict the type of display the component can have? (e.g. `block`, `inline`, `none`, `flex`, `grid`, `inline-flex`, `inline-grid`, and `flow-root`).

References:

* [Accessibility concerns about `display: contents`](https://developer.mozilla.org/en-US/docs/Web/CSS/display#display_contents)
* [CSS Tricks - Display](https://css-tricks.com/almanac/properties/d/display/)
* [CSS Tricks - Two-Value Display Syntax (If anyone interested 🤓)](https://css-tricks.com/two-value-display-syntax-and-sometimes-three/)

https://github.com/dxc-technology/halstack-react/pull/566#event-5978901226

**Comment by GomezIvann** [comment, marked as answer]
Box component doesn't exist anymore, we can close this.
